src/objects/Mesh.js

```diff
@@ -162,7 +162,7 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		function checkIntersection( object, material, raycaster, ray, pA, pB, pC, point ) {
 
 			var intersect;
-
+			if(!material) return null;
```

Comment:
I'm not sure this line makes sense. Is `checkIntersection()` ever be called without a material instance?

Reply:
It happens when mesh.group.materialIndex=-1,

Reply:
This case should never happen. In general we do not check such special cases in the library.

Reply:
I did experience this during the development process